website/screens/components/alert/usage/examples/content.ts

```diff
@@ -0,0 +1,30 @@
+import { DxcAlert, DxcInset, DxcStack } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+  return (
+    <DxcInset space="large">
+      <DxcStack gutter="large">
```

Comment:
Maybe add an `align="center"` to center the example?

Reply:
Added `align=center`.